frontend/src/modules/entitydetails/components/Helpers.js

```diff
@@ -45,16 +43,19 @@ export default class Helpers extends React.Component<Props> {
             locale,
             machinery,
             otherlocales,
-            orderedOtherLocales,
-            preferredLocalesCount,
             parameters,
             user,
             updateEditorTranslation,
             searchMachinery,
         } = this.props;
 
+        if (!otherlocales.translations) {
+            return null;
+        }
```

Comment:
Supernit: you could move this under declarations and then use `if (!translations) {`. :)

Reply:
I actually already tried that but I get errors from Flow about `translations` missing in null or undefined.

Reply:
Oh. Right. Mr Flow. :/ Let's keep it, then.